src/test/java/core/basesyntax/service/ReportServiceTest.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ReportServiceTest {
+    private static ReportService reportService;
+
+    @BeforeAll
+    static void beforeAll() {
+        reportService = new ReportService();
+    }
+
+    @Test
+    void generateReport_StorageEmpty_Ok() {
+        Assertions.assertTrue(reportService.generateReport().isBlank());
+    }
+
+    @Test
+    void generateReport_StorageFilled_Ok() {
```

Comment:
```suggestion
    void generateReport_storageFilled_ok() {
```
let's always use lowercase for the start of each part of the method name separated by underscore, consistency is important